kolena/_experimental/object_detection/evaluator_single_class.py

```diff
@@ -163,11 +167,8 @@ def compute_test_case_metrics(
         recall = compute_recall(tp_count, fn_count)
         f1_score = compute_f1_score(tp_count, fp_count, fn_count)
 
-        average_precision = 0.0
-        if precision > 0:
-            baseline_pr_curve = compute_pr_curve(all_bbox_matches)
-            if baseline_pr_curve is not None:
-                average_precision = compute_average_precision(baseline_pr_curve.y, baseline_pr_curve.x)
+        if average_precision is None:
+            average_precision = compute_average_precision([precision], [recall])
```

Comment:
I'm not sure if I follow this change -- why are we attempting to compute AP with a single `([precision], [recall])` point? Is this a previously missed edge case?